core/cap-0034.md

```diff
@@ -49,6 +52,80 @@ dependent transactions would then perform operations which the smart contract
 had intended to be performed only if `T` had been executed. This would
 potentially be arbitrarily bad for the smart contract.
 
+So here are the conditions under which the above race occurs, and the semantics
+of the current and proposed protocols differ:
+
+1. A smart-contract client submits a series of transactions `T1`, `T2` with
```

Comment:
Same confusion around the word "executed". But how does this enumeration add value beyond the enumeration earlier in the section? They are essentially describing the same thing, right? If you really want to include something like this, you might rework it (probably in the rationale) as a simple concrete example that suffers from this issue but is fixed by this proposal.

Reply:
I agree; the second, more detailed, enumeration made the first one redundant.  I've removed the first one.

I moved the remaining detailed enumeration to a new sub-section of the "Design Rationale", and put in the "Motivation", where that enumeration used to be, a single paragraph in which I try to specify exactly what that problem is without getting into how it comes about.